app/frontend/src/service/user.ts

```diff
@@ -56,9 +56,6 @@ export async function passwordLogin(
 
 /**
  * Login user using a login token and returns session token
- *
- * @param {string} token
- * @returns {Promise<string>}
```

Comment:
Is this removal because the analyzer tells you types anyway?

Reply:
No, because it is written in the function declaration. So for practical purposes: your IDE knows the types anyway.